server.js

```diff
@@ -0,0 +1,37 @@
+const express = require('express'); // importing a CommonJS module
+const helmet = require('helmet');
+
+const usersRouter = require('./data/users/users-router.js')
+
+const postsRouter = require('./data/posts/posts-router.js')
+
+const server = express();
+
+function upperCaser (req, res, next) {
+    if (req.body.hasOwnProperty('name')) {
```

Comment:
Wow, great job on your middleware. You could also put this in it's own folder/file and import into the file you need it in.